### PR TITLE
Verify img to compare size

### PR DESCRIPTION
**Pull request by badb**
Change name of the function to something that better describes what it does. 
Create different implementation for general rendering and for Blender and move it to external file from imgrepr.

### CODE REVIEW

File: tests/apps/rendering/resources/test_imgcompare.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+from apps.rendering.resources.imgcompare import check_size
+
+from golem.testutils import TempDirFixture
+
+
+class TestCompareImgFunctions(TempDirFixture):
+    def test_check_size(self):
+        file1 = self.temp_file_name('img.png')
+        for y in range(100, 200):
```
Comment (ckcollab) [inline tests/apps/rendering/resources/test_imgcompare.py]: Apologies if I'm intruding, but: why this? Why not edge cases like lower bound, middle, upper bound? Won't this just slow down tests needlessly?

Comment (badb) [reply tests/apps/rendering/resources/test_imgcompare.py]: Thanks for pointing this out. Yes, it's unnecessary, I'll fix it.

**Review by banasiakadam60 — APPROVED** [review]
